odoo_initializer/utils/config.py

```diff
@@ -22,6 +22,10 @@ def __init__(self):
             self.db_name = odoo.tools.config["db_name"]
         except KeyError:
             pass
+        try:
+            self.checksum_folder = odoo.tools.config["initializer_checksums"]
```

Comment:
I think we should follow the existing naming pattern and call this property `initializer_checksums_path` instead.

Reply:
refactoring applied.